runtime/resolvers/sql.go

```diff
@@ -32,6 +33,7 @@ type sqlResolver struct {
 type sqlProps struct {
 	Connector string `mapstructure:"connector"`
 	SQL       string `mapstructure:"sql"`
+	Limit     string `mapstructure:"limit"`
```

Comment:
If you change `mapstructure.Decode` to `mapstructure.WeakDecode`, then you can make this an `int` and `mapstructure` will handle the conversion (if needed):
```suggestion
	Limit     int `mapstructure:"limit"`
```

That's better because then we won't error if the caller actually does the right thing and passes an `int` in the first place.